content/en/platform/corda/4.8/open-source/key-concepts-notaries.md

```diff
@@ -21,74 +21,42 @@ title: Notaries
 ## Summary
 
 * *Notary clusters prevent “double-spends”*
```

Comment:
Having now read the full article, I think it would be best to define what a notary is (the concept) vs what the notary cluster is (the physical infrastructure) and use accordingly throughout.